platform/linux-generic/pktio/dpdk.c

```diff
@@ -645,6 +645,7 @@ static inline int mbuf_to_pkt(pktio_entry_t *pktio_entry,
 			 "%d/%" PRIu16 " allocated\n", num, mbuf_num);
 		for (i = num; i < mbuf_num; i++)
 			rte_pktmbuf_free(mbuf_table[i]);
```

Comment:
The debug message could now be removed. Also, `rte_pktmbuf_free()` could be replaced by `rte_pktmbuf_free_bulk()`.